### DISCUSSION TITLE
Fluidd Layout Transfer

### DISCUSSION THREAD

**Discussion by hantrax** [fluidd-core/fluidd#1540] [Q&A] [answered]
Hi Guys..
there is a way to transfer FLUIDD Layout setting to other rpi?

**Comment by pedrolamas** [comment, marked as answer]
You can export all Fluidd settings (including the dashboard layout) from the Settings page and then import those in another Fluidd instance!

![image](https://github.com/user-attachments/assets/b9491aeb-dfc4-4790-b22f-ac606a9de517)

**Comment by hantrax** [comment]
Thanks...
and if I am unable to access to the web interface, is there a way to copy settings from any folders?

**Reply by pedrolamas** [reply]
you can always just copy the moonraker database (the folder `~/printer_data/database`), just make sure to stop Moonraker before copying the files over

**Comment by hantrax** [comment]
Thanks so much!!

it's works for me..
